tests/unit/test_cli_flags.py

```diff
@@ -105,6 +105,14 @@ def test_anonymous_usage_state(
         flags = Flags(run_context)
         assert flags.SEND_ANONYMOUS_USAGE_STATS == expected_anonymous_usage_stats
 
+    def test_resource_types(self, monkeypatch):
+        monkeypatch.setenv("DBT_RESOURCE_TYPES", "model")
+        build_context = self.make_dbt_context("build", ["build"])
+        # Is there any point to this?
```

Comment:
I think so! In terms of setting up the input state of Flags for this test, it seems necessary.

Reply:
Yeah, probably. The behavior of Flags is a bit more opaque than I would like. It could use some heavy commenting/documenting. I'll remove the comment.